taurus/util/tests/test_substitute_objects.py

```diff
@@ -19,8 +19,19 @@ def test_dictionary_substitution():
              (proc_link.scope.lower(), proc_link.id): proc}
 
     test_dict = {LinkByUID.from_entity(proc): LinkByUID.from_entity(mat)}
-    substitute_objects(test_dict, index)
-    assert test_dict[proc] == mat
```

Comment:
Since the resulting process run in `subbed` is not the literal exact same object anymore, this assertion as it was failed (we don't implement a reasonable hash function for our models).